README.md

```diff
@@ -24,4 +35,41 @@ yarn e2e
 yarn test
 ```
 
+### Build Issues
+
+#### node-sass
+One of the development dependencies, node-sass, has an involved build process because it has native components. Node-sass allows Node to compile [SASS style sheets](https://sass-lang.com) into traditional CSS.
+
+If there is not an appropriate set of node-sass/node-gyp binaries for your platform for the configured node-sass/node-gyp version in package.json, the build process will attempt to compile an appropriate version. This doesn't always go well.  
+
+Compiling node-sass (specifically to compile a [node-gyp](https://github.com/nodejs/node-gyp) dependency within node-sass) requires having two main tools. More information is available on the [node-gyp build process](https://github.com/nodejs/node-gyp). 
+* python (Version 2)
+* Build tools for C for your platform. 
+
+Python version 2 must be available on your build machine as `python2`.
+
+The build tools will depend on you OS. See [node-gyp build process](https://github.com/nodejs/node-gyp) for specifics.
+
+##### Workaround if you can't get node-sass to build.
```

Comment:
Good point. I'll play around with a way to rig it up to prevent accidental changes. I had tried going that route at one time but the yarn build complains about a missing dep (maybe from sass-loader). There was a lot of thrashing at the time to get something to work so I'll take a look at that part again in more detail.

Reply:
Surprisingly, just windows 10 home on an i7 chip.